src/Symfony/Component/Security/Http/Firewall/LogoutListener.php

```diff
@@ -87,7 +87,7 @@ public function authenticate(RequestEvent $event)
         $this->eventDispatcher->dispatch($logoutEvent);
 
         $response = $logoutEvent->getResponse();
-        if (!$response instanceof Response) {
+        if (null === $response) {
```

Comment:
`if (!$response = $logoutEvent->getResponse()) {` would be the best fit for our cs